dart.py

```diff
@@ -48,18 +48,18 @@ def simulate_3d_dart_throw(angle_x, angle_y,
     angle_x += sigma * np.random.randn(1)
     angle_y += sigma * np.random.randn(1)
 
-    angle_x = math.radians(angle_x)
```

Comment:
warning message because of expecting scalar values. 
![image](https://github.com/csml-beach/dart/assets/14257985/0d292d26-16d7-4ba1-a0a0-c2ba1463473f)

fixed by changing math.radians to np.radians as np can handle arrays
![image](https://github.com/csml-beach/dart/assets/14257985/acdfdc31-0cfb-4803-868a-537eda359249)